js-runtime/index.js

```diff
@@ -296,14 +730,14 @@ module.exports = {
   // String opcodes
   catstr:  (a, b) => a.concat(b),
   split:   (a, b) => a.split(b),
-  repstr:  (a, b) => new Array(b).fill(a).join(''),
+  repstr:  (a, b) => new Array(parseInt(b.toString())).fill(a).join(''),
```

Comment:
Various random opcodes that take `int64` inputs need to convert to JS `Number`s to actually use them inside of the runtime. Technically this means that some program behaviors that are valid in the AVM do not work in the JS runtime, but generally for things that aren't *practically* valid in the AVM, either, because they involve arrays greater than 2^53 elements long, so I'm not worried about this for the next, uh, century?